### PR TITLE
Improve commands to apply template for collecting and storing Kubernetes events

### PR DESCRIPTION
**Pull request by mburke5678**
https://bugzilla.redhat.com/show_bug.cgi?id=1886519

### CODE REVIEW

File: modules/cluster-logging-eventrouter-deploy.adoc [hunk 1]
```diff
@@ -130,14 +130,14 @@ parameters:
...
-$ oc process -f <templatefile> | oc apply -f -
+$ oc process -f <templatefile> | oc apply -n openshift-logging -f -
 ----
 +
 For example:
 +
 [source,terminal]
 ----
-$ oc process -f eventrouter.yaml | oc apply -f -
+$ oc process -f <templatefile> | oc apply -n openshift-logging -f -
```
Comment (r2d2rnd) [inline modules/cluster-logging-eventrouter-deploy.adoc:140]: This line:

 $ oc process -f <templatefile> | oc apply -n openshift-logging -f -

it should be:

  $ oc process -f eventrouter.yaml | oc apply -n openshift-logging -f -

**Review by gkarager — COMMENTED** [review]
Looks good!

**Pull request merged**